core/node/eth_sender/src/eth_fees_oracle.rs

```diff
@@ -84,6 +105,12 @@ impl GasAdjusterFeesOracle {
                 priority_fee_per_gas,
                 (previous_sent_tx.priority_fee_per_gas * 6) / 5 + 1,
             );
+
+            // same for base_fee_per_gas but 10%
+            base_fee_per_gas = max(
+                base_fee_per_gas,
+                previous_sent_tx.base_fee_per_gas + (previous_sent_tx.base_fee_per_gas / 10) + 1,
+            );
```

Comment:
the `previous_sent_tx.base_fee_per_gas + (previous_sent_tx.base_fee_per_gas / 10) + 1` might have chance to exceed the cap I think, maybe move the logic after this check or check it again in here?

Reply:
we may exceed the cap but only 10% so its not a big issue